xml/System/Math.xml

```diff
@@ -6124,10 +6124,11 @@ The following example demonstrates how to use the <xref:System.Math.Min%2A> meth
         <summary>Returns a specified number raised to the specified power.</summary>
         <returns>The number <paramref name="x" /> raised to the power <paramref name="y" />.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- The following table indicates the return value when various values or ranges of values are specified for the `x` and `y` parameters. For more information, see <xref:System.Double.PositiveInfinity?displayProperty=nameWithType>, <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>, and <xref:System.Double.NaN?displayProperty=nameWithType>.  
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The following table indicates the return value when various values or ranges of values are specified for the `x` and `y` parameters. For more information, see <xref:System.Double.PositiveInfinity?displayProperty=nameWithType>, <xref:System.Double.NegativeInfinity?displayProperty=nameWithType>, and <xref:System.Double.NaN?displayProperty=nameWithType>.
 
 | x                              | y                                                         | Return value                   |
```

Comment:
@tannergooding Lines 6135, 6136, and 6140 don't seem to correspond to anything in the table you added [here](https://github.com/dotnet/dotnet-api-docs/pull/9571#issuecomment-1907387055).

Reply:
6136 corresponds to the first line in the table `pow (x, ±0) is 1 if x is not a signaling NaN`, 6135 then explicitly documents the behavior for NaN.

6140 looks to be trying to do the same thing. It's effectively documenting the counterpart to 6139

-----

In general there are some places where this table is potentially confusing because it lists `0` instead of `+/-0` (or `±0`, if Unicode characters are safe to use). This may lead users to believe this means only positive zero. The same goes for listing `NaN` rather than `qNaN` (quiet NaN) or `sNaN` (signalling NaN).

Notably, however, .NET doesn't differentiate between `qNaN` and `sNaN` itself, which is what ultimately makes some of the entries "redundant".

I'd think the simplest thing would be to be explicit when a difference exists and matters (`±0`) and don't list the difference where it doesn't matter for .NET (`qNaN` vs `sNaN`). A small note could then be left at the bottom explaining that .NET doesn't differentiate between them and cite the relevant portion of the IEEE 754 spec that allows this (most notably its caused by disabling IEEE 754 floating-point exceptions).

Reply:
@tannergooding Another question, how should I show a return value of ±∞ in the docs?

Reply:
I think this is a question for @gewarren. I'm not sure as to the recommended way to do this for the docs.